doc/moduleAT.dox

```diff
@@ -0,0 +1,457 @@
+  \code ./imageProcessing/at-u0-v1 -i ../imageProcessing/Images/triple-spirale-b02.pgm -o tb2 -a 0.162 --lambda 0.0011 --epsilon-1 1.0 --epsilon-2 0.25\endcode
+  </td>
+</tr>
+</table>
+</center>
+
+
+@section AT4 Use of AT models for inpainting
+
+The AT20 model is much superior to the AT01 model for inpainting since
+it is less sensitive to L1-metrication artefacts. Indeed, for large
+\f$ \varepsilon \f$, the discontinuity set is averaged along points,
+not along edges. Hence AT20 is smoothed in all directions while AT01
+is smoothed along axes. This is illustrated on the inpainting
+quadruple angle below. AT01 finds the solution in the L1-metric sense
+for the length of discontinuities, while AT20 finds the correct two
+triple points in the L2-metric sense.
+
+
+<center>
+<table>
+<tr>
+  <td> @image html images/q2-d2-g-mask.png </td>
+  <td> @image html images/q2-d-a0_10000-l0_0620921-u.png </td>
+  <td> @image html images/q2-d2-a1.00000-l0.0100000-u2-v0.png </td>
+</tr>
+<tr>
+  <td> @image html images/q452-d-g-mask.png </td>
+  <td> @image html images/q452-d-a0_10000-l0_0289664-u.png </td>
+  <td> @image html images/q452-d-a1.00000-l0.0250000-u2-v0.png </td>
+</tr>
+<tr align=center>
+  <td> Input \a g + mask </td>
+  <td> Inpainted \a u with AT01 </td>
+  <td> Inpainted \a u with AT20 </td>
+</tr>
+<tr>
+  <td>Command lines are:</td>
+  <td align=center colspan="2">
+  \code ./imageProcessing/at-u0-v1 -i ../imageProcessing/Images/quadruple-color.ppm -o q -m ../imageProcessing/Images/mask-big-disc.pgm -a 1.0 --lambda 0.01 --epsilon-1 4.0 --epsilon-2 0.25\endcode
+  \code ./imageProcessing/at-u0-v1 -i ../imageProcessing/Images/quadruple-45-color.ppm -o q -m ../imageProcessing/Images/mask-disc.pgm -a 1.0 --lambda 0.025 --epsilon-1 4.0 --epsilon-2 0.25\endcode
+
+  \code ./imageProcessing/at-u2-v0 -i ../imageProcessing/Images/quadruple-color.ppm -o q -m ../imageProcessing/Images/mask-big-disc.pgm -a 1.0 --lambda 0.01 --epsilon-1 4.0 --epsilon-2 0.25\endcode
+  \code ./imageProcessing/at-u2-v0 -i ../imageProcessing/Images/quadruple-45-color.ppm -o q -m ../imageProcessing/Images/mask-disc.pgm -a 1.0 --lambda 0.025 --epsilon-1 4.0 --epsilon-2 0.25\endcode
+  </td>
+</tr>
+</table>
+</center>
+
+
+
+
+@section References
+
+[1] Luigi Ambrosio, and Vincenzo Maria Tortorelli. "Approximation of
+functional depending on jumps by elliptic functional via t‐convergence."
+Communications on Pure and Applied Mathematics 43.8 (1990): 999-1036.
+
+[2] Marion Foare, Jacques-Olivier Lachaud, and Hugues Talbot. "Image restoration
+and segmentation using the Ambrosio-Tortorelli functional and discrete calculus."
+In Proceedings of the IAPR International Conference on Pattern Recognition
+(ICPR2016), Cancun, Mexico, 2016.
+
+[3] Marion Foare, Jacques-Olivier Lachaud, and Hugues Talbot. "Numerical
+implementation of the Ambrosio-Tortorelli functional using discrete
+calculus and application to image restoration and inpainting." ????
```

Comment:
You can complete since accepted ;)
To appears to the first Workshop on Reproducible Reaseach In Pattern Recognition (RRPR 2016), Springer LNCS